Airgap Doc Improvements (#743)
astronomer/docs: software/install-airgapped.md

You can also set up your own registry using a dedicated registry service such as [JFrog Artifactory](https://jfrog.com/artifactory/). Regardless of which service you use, follow the product documentation to configure a private registry according to your organization's security requirements.

## Step 2: Fetch Images from Astronomer's Helm Template

> **Note**: You should set up a `config.yaml` before running the below steps. Example `config.yaml` files are in the [AWS](install-aws-standard.md), [Azure](install-azure-standard.md), or [GCP](install-gcp-standard.md) install guides

Reviewer: ```suggestion
**Note**: Astronomer recommends creating a YAML configuration file before completing this task. Sample `config.yaml` files are available here: [AWS](install-aws-standard.md), [Azure](install-azure-standard.md), [GCP](install-gcp-standard.md).
```
Question: should this information be added to the Prerequisites section?

> **Note**: If you utilize other features such as [third party ingress controllers](./third-party-ingress-controllers.md), you made need to synchronize other images not listed with this snippet

Reviewer: ```suggestion
**Note**: If you utilize features such as [third party ingress controllers](./third-party-ingress-controllers.md), you may need to synchronize other images not listed here.
```

The images and tags which are required for your Software installation depend on the version of Astronomer you're installing. 

> Image tags are subject to change, even within existing versions, for example to resolve critical security issues, and therefore not listed here. 

Reviewer: ```suggestion
Image tags are subject to change, even within existing versions. 
```
